tests/integration/requirements.txt

```
@@ -18,4 +18,5 @@ ntnx-vmm-py-client==4.0.1
 ntnx-volumes-py-client==4.0.1
 ntnx-dataprotection-py-client==4.0.1
 ntnx_datapolicies_py_client==4.0.1
-ntnx_lifecycle_py_client==4.0.1
\ No newline at end of file
+ntnx_lifecycle_py_client==4.0.1
+ntnx-objects-py-client>=4.0.1
```

As this will be used for integration testing, we don't need any conflicts here.
```suggestion
ntnx-objects-py-client==4.0.1
```